db/migrate/20240208134335_add_payment_url_to_made_live_forms.rb

```diff
@@ -0,0 +1,18 @@
+class AddPaymentUrlToMadeLiveForms < ActiveRecord::Migration[7.1]
+  def change
+    reversible do |direction|
+      MadeLiveForm.find_each do |made_live_form|
```

Comment:
Could you stub an inline model here please: https://jtway.co/data-migrations-with-rails-6fa3d66f128f

Reply:
Done, I think this is the structure used